### DISCUSSION TITLE
which document is newer/latest, https://g6.antv.antgroup.com  OR  https://antv-g6.gitee.io?

### DISCUSSION THREAD

**Discussion by roccho-dev** [antvis/G6#4508] [Q&A] [answered]
which document is newer/latest, https://g6.antv.antgroup.com  OR  https://antv-g6.gitee.io?

If both documents are the same, no problem, but these are different. 
Please let me know which one I should refer to in the future.

**Comment by hustcc** [comment, marked as answer]
Use：https://g6.antv.antgroup.com/
